cms/migrations/0038_alter_page_site.py

```diff
@@ -6,6 +6,21 @@
 import cms.models.settingmodels
 
 
+def separate_page_treenode(apps, schema_editor):
```

Comment:
I think we should wrap this inside a transaction and try catch as well, so in-case an exception happens we are able to see wha went wrong and with transaction, it only gets committed if all goes fine. What do  you think?

Reply:
Hold on, I just realize, Django does that automatically:; https://docs.djangoproject.com/en/5.1/topics/migrations/#transactions

Reply:
Ah, I didn't knew it did that automatically. I would always do it like this in the migrations to prevent inconsistencies in data if something crash during migration.